src/client/src/components/abstract/sidebar/sidebarLinks/index.jsx

```diff
@@ -7,6 +7,7 @@ import "./style.css";
 class SideLinks extends Component {
   logout = dispatch => {
     removeCookies();
+    localStorage.clear();
     dispatch({ type: "LOGIN_USER", payload: { logging: false } });
```

Comment:
It might be helpful to pull this logic (clear cookies, clear local storage, dispatch) out into its own function, in a separate file.  Then you can reuse that function when you are handling an authentication failure in some other network request.